packages/cloud-cognitive/src/global/js/utils/story-helper.js

```diff
@@ -19,6 +19,7 @@ const storybookStructure = {
   BreadcrumbWithOverflow: `${lib}/$rci/$comp`,
   ButtonMenu: `${lib}/$rci/$comp`,
   ButtonSetWithOverflow: `${lib}/$rci/$comp`,
+  Cascade: `${lib}/$rci/$comp`,
```

Comment:
Oops -- this mechanism has been changed. The storybook structure is now in `/packages/core/storybook-structure.js`. Also, we will need to figure out where it should go in the new structure! @andrea-gm FYI. Is this currently described in the PAL site? For now, maybe you could add it into the 'Internal' section while we figure it out -- something like this:
![image](https://user-images.githubusercontent.com/6385315/130161479-ab6d1db2-bacd-4425-be5c-b33d72aba009.png)